docs/src/modules/accounts.md

```diff
@@ -9,449 +9,29 @@ contract TestMyContract is Test {
-
-
-```solidity
-function sign(uint256 privKey, bytes32 digest) internal pure returns (uint8, bytes32, bytes32);
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`privKey`|`uint256`|The private key to use for signing the digest.|
-|`digest`|`bytes32`|The message digest to sign.|
-
-**Returns**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`<none>`|`uint8`|A tuple containing the signature parameters `(v, r, s)` as a `uint8`, `bytes32`, and `bytes32`, respectively.|
-|`<none>`|`bytes32`||
-|`<none>`|`bytes32`||
-
-
-### derive
-
-*Derives the Ethereum address corresponding to the provided `privKey`.*
-
-
-```solidity
-function derive(uint256 privKey) internal pure returns (address);
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`privKey`|`uint256`|The private key to use for deriving the Ethereum address.|
-
-**Returns**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`<none>`|`address`|The Ethereum address derived from the provided private key.|
-
-
-### deriveKey
-
-*Derives the private key corresponding to the specified `mnemonicOrPath` and `index`.*
-
-
-```solidity
-function deriveKey(string memory mnemonicOrPath, uint32 index) internal pure returns (uint256);
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`mnemonicOrPath`|`string`|The mnemonic or derivation path to use for deriving the private key.|
-|`index`|`uint32`|The index of the derived private key to retrieve.|
-
-**Returns**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`<none>`|`uint256`|The private key derived from the specified mnemonic and index as a `uint256` value.|
-
-
-### deriveKey
-
-*Derives the private key corresponding to the specified `mnemonicOrPath`, `derivationPath`, and `index`.*
-
-
-```solidity
-function deriveKey(string memory mnemonicOrPath, string memory derivationPath, uint32 index)
-    internal
-    pure
-    returns (uint256);
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`mnemonicOrPath`|`string`|The mnemonic or derivation path to use for deriving the master key.|
-|`derivationPath`|`string`|The specific derivation path to use for deriving the private key (optional).|
-|`index`|`uint32`|The index of the derived private key to retrieve.|
-
-**Returns**
+		DAI dai = DAI(0x6B175474E89094C44Da98b954EedeAC495271d0F);
 
-|Name|Type|Description|
-|----|----|-----------|
-|`<none>`|`uint256`|The private key derived from the specified mnemonic, derivation path, and index as a `uint256` value.|
+		// Create an address from a string, mint tokens to the address and impersonate the next call
+		address bob = accounts.create("Bob").mintToken(address(dai), 1337).impersonateOnce();
 
+		// There is no need to `create` an address
+		address charlie = address(0x01).setNonce(10).setBalance(1e18);
 
-### rememberKey
+		DAI daiClone = DAI(address(0x02));
```

Comment:
This might be a bit misleading because it only sets the code but not the state. Maybe line 31 and 33 could be swapped? (first write storage and then read it)